src/web/wfs/src/test/java/org/geoserver/wfs/web/WFSAdminPageTest.java

```diff
@@ -69,6 +76,16 @@ public void testApply() throws Exception {
         // value was updated
         wfs = getGeoServerApplication().getGeoServer().getService(WFSInfo.class);
         assertEquals("testValue1 = 100", 100, (int) wfs.getMaxNumberOfFeaturesForPreview());
+        //test Apply includeWFSRequestDumpFile
+        tester.startPage(WFSAdminPage.class);
+        ft = tester.newFormTester("form");
+        ft.setValue("includeWFSRequestDumpFile", true);
+        ft.submit("apply");
+        // did not switch
+        tester.assertRenderedPage(WFSAdminPage.class);
+        // value was updated
+        wfs = getGeoServerApplication().getGeoServer().getService(WFSInfo.class);
+        assertEquals("includeWFSRequestDUmpFile = true", true, wfs.getIncludeWFSRequestDumpFile());
```

Comment:
Should have been:

```suggestion
        assertTrue("includeWFSRequestDumpFile= true", wfs.getIncludeWFSRequestDumpFile());
```

(a PMD check will likely fail this line, after you get formatting going)